diesel_cli/src/cli.rs

```diff
@@ -222,3 +252,19 @@ fn migration_dir_arg<'a, 'b>() -> Arg<'a, 'b> {
         .takes_value(true)
         .global(true)
 }
+
+fn migration_revert_number_validator(val: String) -> Result<(), String> {
```

Comment:
The help should somehow mention that `all` is also a valid value for `diesel revert -n`